library/src/scripts/events/EventDetails.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import React from "react";
+import { eventsClasses } from "@library/events/eventStyles";
+import { IEvent } from "@library/events/Event";
+import { DataList, IData } from "@library/dataLists/dataList";
+import { IUserFragment } from "@library/@types/api/users";
+import DateTime, { DateFormats } from "@library/content/DateTime";
+import { t } from "@vanilla/i18n/src";
+import { dummyEventDetailsData } from "@library/dataLists/dummyEventData";
+import { renderToString } from "react-dom/server";
+import { ButtonTabs } from "@library/forms/buttonTabs/ButtonTabs";
+import ButtonTab from "@library/forms/buttonTabs/ButtonTab";
+import { EventAttendance } from "@library/events/eventOptions";
+import UserContent from "@library/content/UserContent";
+import { EventAttendees } from "@library/events/Attendees";
+import Heading from "@library/layout/Heading";
+
+export interface IEventExtended extends IEvent {
+    organizer: string;
+    about?: string;
+    going?: IUserFragment[];
+    maybe?: IUserFragment[];
+    notGoing?: IUserFragment[];
+}
+
+/**
+ * Component for displaying an event details
+ */
+export function EventDetails(props: IEventExtended) {
+    const classes = eventsClasses();
+
+    const startDate = <DateTime {...dummyEventDetailsData.dateStart} type={DateFormats.EXTENDED} />;
+    const endDate = <DateTime {...dummyEventDetailsData.dateEnd} type={DateFormats.EXTENDED} />;
+
+    const eventMetaData = [
+        {
+            key: t("When"),
+            value: (
+                <span
+                    dangerouslySetInnerHTML={{
+                        __html: `${renderToString(startDate)}${
+                            dummyEventDetailsData.dateEnd ? ` - ${renderToString(endDate)}` : ""
+                        }`,
+                    }}
```

Comment:
Same note about the other dangerously set HTML.

The code also seems duplicated. Maybe it should be it's own component?